api/proto/info/info.proto

```diff
@@ -211,6 +211,11 @@ message DataSecAtRest {
     repeated VaultInfo vaultList = 3; // per-Vault Info
 }
 
+message ZInfoConfigItems {
+    map <string, string> keyValueMap = 1;
```

Comment:
One of the dangers of doing things piecemeal is that one has to guess what the final shape will be.
I'm guessing we have two types of errors:
1. An issue with the value for a particular item.
2. A configItem key which is unknown.

My take is that #1 is best reported for the particular key i.e., in this map, where the content of the map is { currentValue, error.}
#2 can be reported using the errors you have.

Reply:
For (1) -- The Key / Value that are shown are actually part of the string. What u r suggesting is - Make that actually structured.. Rather than unstructured... 

Hmm.. Make sense.. Will do.

Reply:
I'm suggesting adding an error so that it is a map from string with the key, to a struct/message with a string value and a string error.